iocore/hostdb/HostDB.cc

```diff
@@ -327,8 +466,6 @@ HostDBCache::start(int flags)
   REC_ReadConfigInteger(hostdb_max_size, "proxy.config.hostdb.max_size");
   // number of partitions
   REC_ReadConfigInt32(hostdb_partitions, "proxy.config.hostdb.partitions");
-  // how often to sync hostdb to disk
-  REC_EstablishStaticConfigInt32(hostdb_sync_frequency, "proxy.config.cache.hostdb.sync_frequency");
```

Comment:
This change results in hostdb_sync_frequency always being 0, so no host.db file is ever written.